### PR TITLE
Update HINDI.md

### PR DESCRIPTION
**Pull request by akkshayTandon**
- made changes to HINDI.md file
   - fixed broken links and language errors

### CODE REVIEW

File: translations/README/HINDI.md [hunk 1]
```diff
@@ -49,7 +49,7 @@
 
 | [अरबी](अनुवाद/README/ARABIC.md) | [फ्रेंच](अनुवाद/README/FRENCH.md) | [जर्मन](अनुवाद/रीडमे/जर्मन.एमडी) | [इतालवी](अनुवाद/रीडमे/इटालियन.एमडी) | [कोरियाई](अनुवाद/README/KOREAN.md) | [पोलिश](अनुवाद/README/POLISH.md) | [जापान](अनुवाद/README/JAPANESE.md) |
 | -------------------------------------------------------------- | ------------------------------------------------------- | ------------------------------------------------------- | ------------------------------------------------------- | ------------------------------------------------------ | ------------------------------------------------------ | ------------------------------------------------------ |
-| [पुर्तगाली](अनुवाद/रीडमी/पुर्तगाली.एमडी) | [सर्बियाई](अनुवाद/README/SERBIAN.md) | [स्पेनिश](अनुवाद/README/SPANISH.md) | [तुर्की](अनुवाद/README/TURKISH.md) | [बांग्ला](अनुवाद/README/BANGLA.md) | अंग्रेज़ी |
+| [पुर्तगाली](अनुवाद/रीडमी/पुर्तगाली.एमडी) | [सर्बियाई](अनुवाद/README/SERBIAN.md) | [स्पेनिश](अनुवाद/README/SPANISH.md) | [तुर्की](अनुवाद/README/TURKISH.md) | [बांग्ला](अनुवाद/README/BANGLA.md) | [अंग्रेजी](https://github.com/Syknapse/Contribute-To-This-Project/blob/master/README.md) |
```
Comment (TenTraicion) [inline translations/README/HINDI.md:52]: Those URL's ain't valid!

**Review by TenTraicion — CHANGES_REQUESTED** [review]

File: translations/README/HINDI.md (hunk 1 above)
Comment (TenTraicion) [reply translations/README/HINDI.md:52]: i  was talking about other links on that table.

**Review by TenTraicion — CHANGES_REQUESTED** [review]

File: translations/README/HINDI.md (hunk 1 above)
Comment (akkshayTandon) [reply translations/README/HINDI.md:52]: Oh! Sorry! I will update it.

Comment (akkshayTandon) [reply translations/README/HINDI.md]: Hey!, updated the broken links to other translations.

**Review by TenTraicion — APPROVED** [review]

**Review by TenTraicion — APPROVED** [review]

**Review by alsonick — APPROVED** [review]

**Review by Syknapse — APPROVED** [review]
If there are no further issues this can be merged 👍

**Review by TenTraicion — CHANGES_REQUESTED** [review]
@akkshayTandon,
Thanks for your contribution. but this changes aren't compatible with our current readme files. Kindly check our updated readme and change this file according to the main readme of our project. also change the links to dynamic ones. You can find examples in Translation folder of our project. Thanks!

**Comment by TenTraicion** [comment]
@akkshayTandon, Kindly response or do the required changes mentioned before. We usually close any PR with no response within 7 Days. so kindly response before June 18, 2023. Thank You!

**Pull request closed without merging**